app/models/user.rb

```diff
@@ -45,25 +47,15 @@ def name
     "#{first_name} #{last_name}"
   end
 
-  def approved_accounts
-    Account.
-      joins('LEFT OUTER JOIN `memberships` ON `memberships`.`account_id` = `accounts`.`id`').
-      where(['memberships.user_id = ? and memberships.approved is true', id])
-  end
-
   def approved_active_accounts
-    approved_accounts.active
+    approved_accounts.try(:active)
   end
 
   def approved_account_stories
-    Story.
-      joins('LEFT OUTER JOIN `memberships` ON `memberships`.`account_id` = `pieces`.`account_id`').
-      where(['memberships.user_id = ? and memberships.approved is true', id])
+    !approved_accounts.nil? ? Story.where(account_id: approved_accounts.try(:ids)) : []
```

Comment:
Could certainly be in there! I'm not quite sure where to place the method definitions for `approved_account_series` and `approved_account_stories` because they are called in the representers, which would lead me to want to put them in the model layer, but also are called in the controller...